torchao/kernel/README.md

```diff
@@ -0,0 +1,19 @@
+## Autotuner and custom Triton kernels
+
+### Use case
```

Comment:
Is intent to fill this out later? Might be better to open an issue if you'd like to do this later